samples/csharp_dotnetcore/82.skills-sso-cloudadapter/RootBot/TokenExchangeSkillHandler.cs

```diff
@@ -39,6 +41,7 @@ public class TokenExchangeSkillHandler : CloudSkillHandler
             BotFrameworkAuthentication auth,
             SkillsConfiguration skillsConfiguration,
             IConfiguration configuration,
+            ConfigurationBotFrameworkAuthentication configurationBotFrameworkAuthentication,
```

Comment:
The constructor is missing the property setting. 
And probably you should add this new parameter when you instantiates the class.
```CSharp
_configurationBotFrameworkAuthentication = configurationBotFrameworkAuthentication ?? throw new ArgumentNullException(nameof(configurationBotFrameworkAuthentication));
```